ml-algorithms/src/main/java/org/opensearch/ml/engine/utils/ScriptUtils.java

```diff
@@ -5,52 +5,36 @@
 
 package org.opensearch.ml.engine.utils;
 
-import com.google.gson.Gson;
-import org.opensearch.ml.common.connector.MLPostProcessFunction;
-import org.opensearch.ml.common.connector.MLPreProcessFunction;
-import org.opensearch.ml.common.utils.StringUtils;
```

Comment:
Not sure why do we have to remove it from here and then use that in the code?

Reply:
Because the Pre/Post process function has nothing to do with painless script, they are java functions, ScriptUtils' single responsibility is to execute customized painless script.

Reply:
I mean we removed import : `import org.opensearch.ml.common.utils.StringUtils;`

then used in the code like: `org.opensearch.ml.common.utils.StringUtils.fromJson(resultJson, "result");`

any advantage on using like this?

Reply:
Oh, this one. I think it's because I was using apache common StringUtils in this class as well so make this a fully qualified name, but apache one was removed, so we can use import StringUtils now.